django/test/utils.py

```diff
@@ -640,3 +640,14 @@ def setUp(self):
 
     def tearDown(self):
         self.logger.handlers[0].stream = self.old_stream
+
+
+def tag(*tags):
+    """
+    Decorator to add tags to test
+    """
+    def decorator(function):
```

Comment:
Btw, is it easy to extend tags idea to modules, like py.test does?
On module level:
`
tags = tag('core')
`